lib/chef/resource/powershell_script.rb

```diff
@@ -21,7 +21,7 @@ class Chef
   class Resource
     class PowershellScript < Chef::Resource::WindowsScript
 
-      set_guard_inherited_attributes(:interpreter)
+      set_guard_inherited_attributes(:interpreter, :use_inline_powershell)
```

Comment:
I think this may be contributed to the Linux test failures in buildkite

Reply:
yeah, it seems so - I'm not sure how I missed that one, since it even tripped locally for me this time... I'm guessing i was dumb and ran the test before the change!

Reply:
looks like updating the test that consumes this fixed this.  woo!  thanks for the heads up!